code/src/Form.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react'
+import { Dropdown } from 'Dropdown'
+import { Radio } from 'Radio'
+import { Text } from 'Text'
+import { Submit } from 'Submit'
+
+
+
+export const Form = () => {
+ 
+    const [dropInput, fetchDropInput] = useState("")
+    const [radioInput, fetchRadioInput] = useState("")
+    const [textInput, fetchTextInput] = useState("")
+    const [section, setSection] = useState('start')
+
```

Comment:
nice for having all the "useState" in one place! 💃

Reply:
Best movie!